pages/services/chat.html

```diff
@@ -19,7 +19,7 @@ <h1 class="display-4">Matrix</h1>
 </div>
 
 <h2 class="pt-4" id="homeserverinfo">Connection Information</h2>
-<p><strong>To register on or connect to our homeserver, simply use <code>https://chat.privacytools.io</code> as the homeserver in your Matrix client.</strong></p>
+<p><strong>To register on or connect to our homeserver, simply use <code>https://chat.privacytools.io</code> as the homeserver in your Matrix client. <p><strong>Please see our community <a href="https://wiki.privacytools.io/view/PrivacyTools:Code_of_Conduct">code of conduct</a> before joining our rooms or using our services.</strong></p>
```

Comment:
```suggestion
<p><strong>To register on or connect to our homeserver, simply use <code>https://chat.privacytools.io</code> as the homeserver in your Matrix client.</strong></p>
<p><mark>You must read and accept our community <a href="https://wiki.privacytools.io/view/PrivacyTools:Code_of_Conduct">code of conduct</a> before joining our rooms or using our services.</mark></p>
```

Fix crazy formatting